airflow/utils/log/es_task_handler.py

```diff
@@ -139,9 +140,17 @@ def es_read(self, log_id, offset):
             .sort('offset')
 
         s = s.filter('range', offset={'gt': offset})
+        max_log_line = s.count()
+        if 'download_logs' in metadata and metadata['download_logs'] and 'max_offset' not in metadata:
+            try:
+                metadata['max_offset'] = s[max_log_line - 1].execute()[-1].offset if max_log_line > 0 else 0
+            except Exception as e:
+                msg = 'Could not get current log size with log_id: {}, ' \
```

Comment:
think we can just pass e to exception and log the msg separately if you want to keep it